code/controllers/subsystem/garbage.dm

```diff
@@ -363,71 +363,78 @@ SUBSYSTEM_DEF(garbage)
 /// Should be treated as a replacement for the 'del' keyword.
 ///
 /// Datums passed to this will be given a chance to clean up references to allow the GC to collect them.
-/proc/qdel(datum/D, force=FALSE, ...)
-	if(!istype(D))
-		del(D)
+/proc/qdel(datum/to_delete, force = FALSE, ...)
```

Comment:
this may make mso upset fair warning

Reply:
i'm not opinionated enough on this tbh. 
should i just put back the single letter vars

Reply:
the rest of ssgarbage is full of em, but that's just cause mso code 
this is good and fine (I'm mostly just preparing for an i told you so him getting upset isn't the end of the world he just disagrees with us about single letter vars)